app/src/ui/suggested-actions/dropdown-suggested-action.tsx

```diff
@@ -155,14 +153,15 @@ export class DropdownSuggestedAction<T extends string> extends React.Component<
             <p className="discoverability">{discoverabilityContent}</p>
           )}
         </div>
-        <DropdownSelectButton<T>
-          selectedValue={value}
-          options={this.props.suggestedActions.map(({ label, value }) => ({
+        <DropdownSelectButton
+          checkedOption={value}
```

Comment:
If you accept my previous (nit) suggestion:
```suggestion
          checkedOption={id}
```